source/server-selection/server-selection.rst

```diff
@@ -458,7 +458,7 @@ with error code 160 (SERVER-24421).
 
 During server selection,
 drivers (but not mongos) MUST raise an error if ``maxStalenessSeconds`` is a positive number,
-and any server's ``maxWireVersion`` is less than 5. [#]_
```

Comment:
We can remove this error checking rule now that we've dropped support for 3.4 (DRIVERS-1914/DRIVERS-1315)

Reply:
I kept the rule but clarified that it only applies if minWireVersion < 5. I noted the rationale in https://github.com/mongodb/specifications/pull/1064/files#r707775526.